composer.json

```diff
@@ -5,10 +5,10 @@
     "keywords": ["laravel", "framework"],
     "license": "MIT",
     "require": {
-        "php": "^8.3",
+        "php": "^8.2.4",
```

Comment:
This change can be revered. We are enforcing environments to use PHP 8.3 everywhere.

Reply:
Latest PHP version on XAMPP is 8.2.4. I will move to Herd for the upcoming tasks. Pushed the change. There should be no conflicts as we still haven't used any of php 8.3 features.